### DISCUSSION TITLE
Frigate sublabels

### DISCUSSION THREAD

**Discussion by YBonline** [blakeblackshear/frigate#4568] [Ideas]
Is it possible to send a a signal of a sublabel to the recordings when an event happens? I have an access control system around the facility I'm monitoring with frigate, and I generally have a camera at each door, along with a card scanner that unlocks the door. The card scanner produces a MQTT event saying "Card 1111 was scanned" and I'd like to have the recording at the time that event is produced receive a sublabel marking that event happened during that recording. How feasible is this/where should I get started?

**Comment by NickM-27** [comment]
See our docs, should be able to use the http API https://docs.frigate.video/integrations/api#post-apieventsidsub_label
